### ISSUE TITLE
Command formation is way too spaced out

### ISSUE THREAD

**Issue by Syncinus** [Anuken/Mindustry-Suggestions#700]
**Describe what you would like changed, and why.**
Right now formation forms a big circle, the problem with this circle is that after a point, it gets much too spread out and units start falling out of range, which is annoying because the old way it worked didn't have that kind of problem (though it did notably have a lot of other problems).

**Describe the changes you want to propose. Include possible alternatives.**
Basically make command form multiple "rings" around the commander based on unit size and range. This will mainly allow for larger command limits to be viable, especially in things like mods.
